pkl-core/src/main/java/org/pkl/core/http/HttpClientBuilder.java

```diff
@@ -67,39 +57,14 @@ public HttpClient.Builder setRequestTimeout(Duration timeout) {
   }
 
   @Override
-  public HttpClient.Builder addCertificates(Path file) {
-    certificateFiles.add(file);
+  public HttpClient.Builder addCertificates(Path path) {
+    certificateFiles.add(path);
     return this;
   }
 
   @Override
-  public HttpClient.Builder addCertificates(URI url) {
-    var scheme = url.getScheme();
-    if (!"jar".equalsIgnoreCase(scheme) && !"file".equalsIgnoreCase(scheme)) {
-      throw new HttpClientInitException(ErrorMessages.create("expectedJarOrFileUrl", url));
-    }
-    certificateUris.add(url);
-    return this;
-  }
-
-  public HttpClient.Builder addDefaultCliCertificates() {
-    var fileCount = certificateFiles.size();
-    if (Files.isDirectory(caCertsDir)) {
-      try (var files = Files.list(caCertsDir)) {
-        files.filter(Files::isRegularFile).forEach(certificateFiles::add);
-      } catch (IOException e) {
-        throw new HttpClientInitException(e);
-      }
-    }
-    if (certificateFiles.size() == fileCount) {
-      addBuiltInCertificates();
-    }
-    return this;
-  }
-
-  @Override
-  public HttpClient.Builder addBuiltInCertificates() {
-    certificateUris.add(getBuiltInCertificates());
+  public Builder addCertificates(byte[] certificateBytes) {
```

Comment:
Calling this `addCertificates` is misleading, as calling this method more than once will just reset the bytes, not add them like `addCertificates(Path)`. Either have a `List<ByteBuffer>` or rename it to `setCertificates`.

Reply:
Changed this to have append semantics.